src/main/java/com/minecolonies/coremod/commands/CommandEntryPointNew.java

```diff
@@ -287,7 +299,7 @@ private static void throwCommandUsageExceptionIfRequiredArgumentsAreNotProvided(
                 {
                     if ((null != executionActionArgument) && actionArgument.getName().equals(executionActionArgument.getName()))
                     {
-                        if (!executionActionArgument.isValueSet())
+                        if (!actionMenuState.isValueSet(executionActionArgument))
                         {
                             if (null == badArgument)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 5 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS134)